From e460c51017a7d1e04511ea4d293296a3d422049a Mon Sep 17 00:00:00 2001
From: Nobuhiro Iwamatsu <iwamatsu@nigauri.org>
Date: Mon, 3 Apr 2017 01:42:26 +0900
Subject: [PATCH] Add config.h in sbc/sbc_primitives.h

When we use "--enable-high-precision" option of configure, this does not work
effectively. Because SBC_HIGH_PRECISION enabled by this option is not available
in "sbc/sbc_primitives.h".
This adds config.h to "sbc/sbc_primitives.h" to use SBC_HIGH_PRECISION.

Signed-off-by: Nobuhiro Iwamatsu <iwamatsu@nigauri.org>
---
 sbc/sbc_primitives.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/sbc/sbc_primitives.h b/sbc/sbc_primitives.h
index e01c957..b060484 100644
--- a/sbc/sbc_primitives.h
+++ b/sbc/sbc_primitives.h
@@ -27,6 +27,10 @@
 #ifndef __SBC_PRIMITIVES_H
 #define __SBC_PRIMITIVES_H
 
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
 #define SCALE_OUT_BITS 15
 #define SBC_X_BUFFER_SIZE 328
 
-- 
2.11.0

